src/frontend/src/lib/utils/info.utils.ts

```diff
@@ -3,7 +3,8 @@ import { browser } from '$app/environment';
 export type HideInfoKey =
 	| 'oisy_ic_hide_bitcoin_info'
 	| 'oisy_ic_hide_ethereum_info'
-	| 'oisy_ic_hide_erc20_info';
+	| 'oisy_ic_hide_erc20_info'
+	| 'oisy_ic_hide_bitcoin_transactions_disclaimer';
```

Comment:
`activity` instead of `transactions_disclaimer` since it's the activity page? No strong opinion though